### PR TITLE
Palo Alto Networks PAN-OS - Added the *SerialNumber* argument to the …

### PR DESCRIPTION
**Pull request by bakatzir**
…***panorama-get-pcap*** command to allow downloading a PCAP file from a Panorama instance

## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/32991
fixes: https://github.com/demisto/etc/issues/27501
relates to: https://github.com/demisto/etc/issues/22835

## Minimum version of Demisto
- [x] 3.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by dproctor-palo** [comment]
@bakatzir the customer has a channel named Apollo-Management on DFIR.  He is Josh Moorhouse - jmoorehouse@apollo.com.  Let me know your preferred method of providing him the fix to test.

**Comment by bakatzir** [comment]
@dproctor-palo this is a public repo, please be mindful here when posting.
feel free to add me to the DFIR channel, i'll send it there.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -815,11 +815,11 @@ def panorama_get_address(address_name: str) -> Dict:
     return result['response']['result']['entry']
 
 
-def panorama_get_address_command(name: str):
+def panorama_get_address_command(args: dict):
     """
     Get an address
     """
-    address_name = name
+    address_name = args.get('name')
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:822]: same question as the tag

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 2]
```diff
@@ -2425,7 +2425,8 @@ Returns information for a Panorama PCAP file. The recommended maximum file size
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| pcapType | Type of Packet Capture. | Required | 
+| pcapType | Type of Packet Capture. | Required |
+| serialNumber | The serial number of the firewall to download the PCAP from. | Optional |
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/README.md:2429]: you added it in 2 commands but I can only see it her once.

**Review by yuvalbenshalom — COMMENTED** [review]
looks good, see my comments

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 3]
```diff
@@ -756,11 +756,11 @@ def panorama_list_addresses(tag: Optional[str] = None):
     return result['response']['result']['entry']
 
 
-def panorama_list_addresses_command(tag: Optional[str] = None):
+def panorama_list_addresses_command(args: dict):
     """
     Get all addresses
     """
-    addresses_arr = panorama_list_addresses(tag)
+    addresses_arr = panorama_list_addresses(args.get('tag'))
```
[the remark this answers is not in the document]
Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:763]: If there will be no args passed to `panorama_list_addresses_command` in `**args`, the code will break. which is a plausible scenario in the TPB.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 1 above)
Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:822]: If there will be no args passed to `panorama_list_addresses_command` in `**args`, the code will break. which is a plausible scenario in the TPB.

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/PAN-OS/ReleaseNotes/1_6_13.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Added the *SerialNumber* argument to the ***panorama-get-pcap***, ***panorama-list-pcap*** commands to allow downloading a PCAP file from a Panorama instance.
```
Comment (amshamah419) [inline Packs/PAN-OS/ReleaseNotes/1_6_13.md:4]:
```suggestion
- Added the *serialNumber* argument to the ***panorama-get-pcap***, ***panorama-list-pcap*** commands to allow downloading a PCAP file from a Panorama instance.
```

**Pull request merged**